### PR TITLE
Fix manual Makefiles and add CI for them

### LINKED ISSUE #108: hidtest-libusb and hidtest-hidraw do not compile
Original signal11 hidtest.cpp was renamed to hidtest.c and example makefiles "Makefile-manual" in hidapi/libusb and hidapi/linux stopped working.

### PR DESCRIPTION
**Pull request by z3ntu**
Fixes #108

### CODE REVIEW

File: .builds/freebsd.yml [hunk 1]
```diff
@@ -16,3 +17,7 @@ tasks:
     cd hidapi
     make
     make DESTDIR=$PWD/root install
+    make clean
+- build-manual: |
+    cd hidapi/libusb
+    gmake -f Makefile-manual
```
Comment (z3ntu) [inline .builds/freebsd.yml]: Unfortunately the freebsd manual makefiles aren't posix compliant, so gmake is needed there. I researched a bit but I didn't find a good reference how e.g. `ifeq` statements should look like in POSIX make, so let's just use gmake

**Comment by Youw** [comment]
LGTM

**Comment by z3ntu** [comment]
bump?

**Comment by Youw** [comment]
I don't this system(s) to try, but the changes looks OK.
If there are any issues, we could fix them later.

**Pull request merged**